test/types/package-lock.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "types",
+    "lockfileVersion": 3,
+    "requires": true,
+    "packages": {
+        "": {
+            "dependencies": {
+                "pdfjs-dist": "../../build/typestest/"
+            }
+        },
+        "../../build/dist": {
+            "name": "pdfjs-dist",
+            "version": "4.0.201",
```

Comment:
Here, and elsewhere below, please either use the *correct* version number (via a build-script) or use a dummy value if that's simpler.

Furthermore, how was this file generated and more importantly how will it be updated to adapt to future library changes (this should ideally be automated via a script)?